actors/miner/src/lib.rs

```diff
@@ -1143,6 +1148,7 @@ impl Actor {
         if proven_manifests.is_empty() {
```

Comment:
I know it is currently unreachable but it might make sense to leave a comment in the aggregate update proof case to correctly construct a batch_gen so whoever implements that gets a reminder.

Reply:
It also might make sense to abstract this proving code from 1112 to 1151 into a separate function.  And then filter the valid_unproven_usis using `successes` as we do elsewhere.

Reply:
It's a good idea, I tried it and think it will work. However it required too much code change to do cleanly here for me to feel comfortable taking your existing approval. Some clunkiness comes from `BatchReturn::successes` returning a vector of _references to_ the underlying, and we end up with something like `Vec<&(&a, &b)>`. More helper code to flatten that isn't too hard, but neither trivial.